features/integration.feature

```diff
@@ -1,16 +1,75 @@
+# These tests only need to run once, because we use an external
+# Conjur server, so we'll tag them to run with the V5 tests (the
+# external server is Conjur V5 Enterprise)
+@conjur-version-5
 @integration
-Feature: Integration Tests
+Feature: Integration Tests for PCF 2.4
 
   Scenario: Service broker functions correctly with PCF 2.4
-  
     Given I login to PCF and target my organization and space
     And I load a secret into Conjur
     And I install the service broker
-    And I create a service instance for Conjur
-    And I load policy to define my org and space
+
+    When I create a service instance for Conjur
+    Then the instance resource exists
+    And the policy for the org and space exists
+
+    When I privilege the org layer to access a secret in Conjur
+    And I privilege the space layer to access a secret in Conjur
+    
+    And I push the sample app to PCF
+    And I privilege the app host to access a secret in Conjur
+    And I start the app
+    Then I can retrieve the secret values from the app
+
+    When I remove the service broker
```

Comment:
should this be `when I remove the service instance`? (that is what deprovision is)